src/platform/forms-system/src/js/widgets/DateWidget.jsx

```diff
@@ -91,12 +98,13 @@ export default class DateWidget extends React.Component {
             id={`${id}Month`}
             value={month}
             disabled={disabled}
+            aria-describedby={`${id}-label ${id}-${'error-message'}`}
```

Comment:
I've included both the label (a `legend` in this case) and the error span - I'm not sure if including the ID here if it doesn't exist is a problem for all screenreaders.

Reply:
I'm on the fence about adding the `aria-describedby` here. The legend is read aloud the first time users enter the group if they're moving by `TAB` keypress, and is read inline if users are moving by text node. The attribute could also throw an axe error if it's present and the ID attribute isn't.

All that to say, I think we'll be better off removing the `aria-describedby` attribute.

Reply:
It should be okay if the error span ID existed, but the error element content was empty?

Reply:
Hmm, actually the error state and the widget are decoupled, so there isn't an easy way to tell if there is an error. I'll go ahead and remove the `aria-describedby` for now.